core/chain/coin/chainTokens.ts

```diff
@@ -921,6 +922,7 @@ export const chainTokens: Partial<
     ...IBC_TOKENS.map(t => ({
       ...t,
       chain: Chain.THORChain,
+      decimals: chainFeeCoin[Chain.THORChain].decimals,
```

Comment:
@rcoderdev can you double check this? I think all tokens can have their own different decimals